tools/nni_cmd/updater.py

```diff
@@ -39,6 +39,22 @@ def validate_file(path):
     if not os.path.exists(path):
         raise FileNotFoundError('%s is not a valid file path' % path)
 
+def validate_dispatcher(args):
+    '''validate if the dispatcher of the experiment supports importing data'''
+    nni_config = Config(get_config_filename(args))
```

Comment:
Suggest to use term "tuner" to replace "dispatcher",  same as above.

Reply:
Done